src/renderer/atlas/AtlasEngine.h

```diff
@@ -1105,6 +1110,7 @@ namespace Microsoft::Console::Render
             bool enableTransparentBackground = false;
 
             std::wstring customPixelShaderPath; // changes are flagged as ApiInvalidations::Device
+            std::wstring customPixelShaderImagePath;

```

Comment:
nit: i like how _each_ line says where changes are flagged; should we just propagate that here too?

```suggestion
            std::wstring customPixelShaderImagePath; // changes are flagged as ApiInvalidations::Device
```